crates/wasmtime/src/config.rs

```diff
@@ -676,7 +675,6 @@ impl Config {
     /// This is `true` by default.
     ///
     /// [proposal]: https://github.com/webassembly/simd
-    /// [relaxed simd proposal]: https://github.com/WebAssembly/relaxed-simd
```

Comment:
I think that this link reference may need to stay?

Reply:
Since there's now a separate method (`wasm_relaxed_simd`) which is specific to relaxed SIMD and references that proposal I thought it was best to remove it from here and leave it referenced in the more specific place. I can add it back in if you'd prefer though?

Reply:
Ah I see, I didn't realise they were acting as references for the above links! I've added it back in.